src/main/java/pl/lodz/p/it/ssbd2023/ssbd02/config/DataSourceConfig.java

```diff
@@ -7,7 +7,7 @@
 import java.sql.Connection;
 
 @DataSourceDefinition(
-    name = "java:app/jdbc/ssbd02admin",
+    name = "java:app/jdbc/postgres/ssbd02admin",
```

Comment:
Actually I don't see any reason to store postgres DataSources if we don't use it :thinking: